jib-core/src/main/java/com/google/cloud/tools/jib/api/MainClassFinder.java

```diff
@@ -42,7 +42,7 @@
   public static class Result {
 
     /** The type of result. */
-    public enum Type {
+    public enum Category {
```

Comment:
Too bad, this is a public API class and the `Type` is exposed as public. We need `@depraecated` and a `CHANGELOG` update?

Reply:
You know what, I didn't even realize this was api. Lemme go reverse this and just use fqn for asm.